benchmarks/railsbench/benchmark.rb

```diff
@@ -19,6 +19,9 @@
   unless success
     raise "Couldn't set up railsbench!"
   end
+
+  # Over time, it's possible to fill a device with logfiles
+  system("rm -f #{__dir__}/log/production.log")
```

Comment:
Wouldn't `File.unlink path if File.exist? path` be cleaner here?
Notably `system` doesn't raise anything if the process fails.

Reply:
Could mean some kind of filesystem error, as you say. Could mean we don't have permissions set up properly. Could mean something's wrong in the shell. Mostly removing the log is a usually-optional action meant for unusual circumstances, so ignoring failure seems reasonable.

Reply:
It's actually quite common to have a logfile on the system. There's normally nothing wrong with that. This is being removed only because we don't want to accumulate logfiles over a lot of runs since that can eventually fill up the disk. That's not going to be as common with the log level turned down -- :info got very large.